src/utils/leela_encodings.py

```diff
@@ -152,10 +152,10 @@ def encode_seq(
     us, them = chess.WHITE, chess.BLACK
     for alg_move in seq.split():
         board_tensors[us].append(board_to_tensor(board, (us, them)))
-        if position_evaluator is not None:
-            position_evaluations[us].append(position_evaluator(board, (us, them)))
         if alg_move.endswith("."):
             continue
```

Comment:
This should be the first test in the loop, right?

Reply:
oh you're right, I was so focused on the position evaluations, I didn't notice that this should go above the board_tensors[us].append() line as well. I'll make a commit for this